src/librustc_metadata/diagnostics.rs

```diff
@@ -26,6 +26,16 @@ name. Example:
 ```
 "##,
 
+E0455: r##"
+A framework kind link was used and the OS utilized to compile isn't the OSX. Erroneous code example:
```

Comment:
Does the running OS really have to be OSX, or just the target we're compiling for?